integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -132,7 +210,22 @@ def clean_content(self) -> str:
                 tostring(monospaced, encoding="unicode"),
             )
 
-        # Set link-external as class for external links
+    def update_links(self, content: HtmlElement) -> None:
```

Comment:
I'm undecided if it is good to first gather all methods for update_links in one method and then call this one. On one hand I think it's cleaner on the other hand I can see myself being confused at first. What do you think? Does anyone have a clear preference?

Reply:
I think both is fine.